### ISSUE TITLE
MacOS app login error

### ISSUE THREAD

**Issue by dpazdan** [keybase/client#5181]
Keybase GUI Version: 1.0.18-20161219185928+7868d49

When I log in I immediately get:
There was an error provisioning

Unknown Error: GPG is unavailable on this device

**Comment by cjb** [comment]
Please could you run `keybase log send` and tell us the ID? Thanks!

CC @patrickxb

**Comment by dpazdan** [comment]
79975c58a3d9411341e3871c

**Comment by cjb** [comment]
@patrickxb I thought macOS shipped gpg by default, any ideas?

**Comment by patrickxb** [comment]
@cjb I don't think it does.  this is the standard package to install:  https://gpgtools.org

@dpazdan you have a PGP key associated with your account:  D667B05C155DE126.  do you still have access to it?

**Comment by dpazdan** [comment]
Thanks I'll give this a try and report back.
